result.rb

```diff
@@ -1,12 +1,355 @@
+puts JSON.pretty_generate(data_arr)
 [
   {
     "name": "2259A3648041",
     "balance": "924,14",
-    "currency": "MDL"
+    "currency": "MDL",
+    "nature": "Salarial Basic MDL ",
+    "transactions": [
+      {
+        "trans_date": "30/07/16",
+        "details": "Retail MDA Chisinau MAIB ROGOB  MARKET",
+        "suma": "-48.79",
+        "currency": "MDL",
+        "date": "01/08/16",
```

Comment:
[first answer](http://stackoverflow.com/questions/10286204/the-right-json-date-format). no need to save timezone and time but date should me YYYY-MM-DD